pkg/scannerv4/mappers/mappers.go

```diff
@@ -357,11 +370,51 @@ func sortBySeverity(ids []string, vulnerabilities map[string]*v4.VulnerabilityRe
 	})
 }
 
-func toProtoV4VulnerabilitiesMap(ctx context.Context, vulns map[string]*claircore.Vulnerability, nvdVulns map[string]map[string]*nvdschema.CVEAPIJSON20CVEItem) (map[string]*v4.VulnerabilityReport_Vulnerability, error) {
+// rhelVulnsEPSS gets highest EPSS score for each rhel advisory name in a given image
```

Comment:
"for each Red Hat advisory" would be more accurate

Also remove the "in a given image" since this function doesn't take in an image